app/services/AppealService.scala

```diff
@@ -125,4 +125,17 @@ class AppealService @Inject()(appealsConnector: PEGAConnector,
       None
     }
   }
+
+  def truncateFilename(fileName: String): String = {
+    logger.info(s"[AppealService][truncateFilename] File name length: ${fileName.length}, truncating to ${appConfig.maximumFilenameLength}")
```

Comment:
This may need to change slightly, for: 'file1.txt' it would say: 'File name length: 9' which is wrong as the file name (without the extension) is 5

Reply:
Also may be worth having a file reference logged so we can trace it